Test/git-issues/git-issue-356.dfy.expect

```diff
@@ -0,0 +1,14 @@
+
+Dafny program verifier finished with 5 verified, 0 errors
+
+Dafny program verifier finished with 0 verified, 0 errors
+END
```

Comment:
Add some `print` statements to (some of) the tests, so we can see that the conversions do something.

Reply:
Done